src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java

```diff
@@ -292,437 +290,434 @@ private ExecutableNormalizedOperation createExecutableNormalizedOperationImplWit
                 rawVariables,
                 options.getGraphQLContext(),
                 options.getLocale());
-        return createNormalizedQueryImpl(graphQLSchema,
+
+        return new ExecutableNormalizedOperationFactoryInner(
+                graphQLSchema,
                 operationDefinition,
-                fragments,
+                getOperationResult.fragmentsByName,
                 coercedVariableValues,
                 normalizedVariableValues,
-                options);
-    }
-
-    /**
-     * Creates a new ExecutableNormalizedOperation for the provided query
-     */
-    private ExecutableNormalizedOperation createNormalizedQueryImpl(GraphQLSchema graphQLSchema,
```

Comment:
This diff is terrible to read, but basically what I did was:
- created `ExecutableNormalizedOperationFactoryInner` and moved `createNormalizedQueryImpl` and all the helper methods into it.
- created instance fields for all the objects that were being passed around to helper methods